### PR TITLE
Migrate Namespace Store Creation Page

### PR DESCRIPTION
**Pull request by bipuladh**

### CODE REVIEW

File: packages/odf/components/namespace-store/create-namespace-store.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        redirectHandler={(resources) => {
+          const lastIndex = resources.length - 1;
+          const resourcePath = `${referenceForModel(
+            NooBaaNamespaceStoreModel
+          )}/${getName(resources[lastIndex])}`;
+          isODF
+            ? history.push(`/odf/resource/${resourcePath}`)
+            : history.push(
+                `/k8s/ns/${ns}/clusterserviceversions/${appName}/${resourcePath}`
+              );
```
Comment (SanjalKatiyar) [inline packages/odf/components/namespace-store/create-namespace-store.tsx:52]: don't think we need it anymore, but we can remove later as well ! whichever is more alright.

File: packages/odf/components/namespace-store/namespace-store-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <ModalBody>
+          <p>
+            {t(
+              'Represents an underlying storage to be used as read or write target for the data in the namespace buckets.'
+            )}
+          </p>
+          <NamespaceStoreForm
+            namespace={namespace}
+            onCancel={() => close()}
+            redirectHandler={() => close()}
```
Comment (SanjalKatiyar) [inline packages/odf/components/namespace-store/namespace-store-modal.tsx:29]:
```suggestion
            onCancel={closeModal}
            redirectHandler={closeModal}
```
shouldn't this be `closeModal` instead ??

**Comment by weirdwiz** [comment]
LGTM
image

File: packages/odf/components/namespace-store/create-namespace-store.tsx (hunk 1 above)
Comment (bipuladh) [reply packages/odf/components/namespace-store/create-namespace-store.tsx:52]: Will do it in the bucketclass PR.

File: packages/odf/components/namespace-store/namespace-store-modal.tsx (hunk 2 above)
Comment (bipuladh) [reply packages/odf/components/namespace-store/namespace-store-modal.tsx:29]: Yes they should be there are other problems in this component as well. I am fixing them as part of create-bc PR as it's easier to work with them when I can actually see them. :)

**Pull request merged**